spec/factories/events.rb

```diff
@@ -19,6 +19,8 @@
 #
 
 FactoryBot.define do
+  sequence(:body) { |n| "Please change line #{n}" }
```

Comment:
What do you think about renaming this sequence to something like `review_body`?

Reply:
This sequence is being used by different factories (review_payload, review_comment_payload and review_comment), so I put it in Event factory, which comes to be a common place in sense of hierarchy.
Should I create a sequence for each one?

Reply:
Ahhh, I see. You're right. Sorry, now I remember we talked about this yesterday.

Maybe for this attribute we do can use Faker, right?

Reply:
Solved in an offline conv.